cjs/interface/css-style-declaration.js

```diff
@@ -82,6 +82,21 @@ class CSSStyleDeclaration extends Map {
     refs.get(this).setAttribute('style', value);
   }
 
+  getPropertyValue(name) {
+    const self = this[PRIVATE];
```

Comment:
For my own understanding/curiosity, what is the difference between `this` and `this[PRIVATE]` (which is a getter that returns `this`)?

It seems like the `cssText` getters and setters are the only ones that use `this`

Reply:
`this` points at the proxy, not at the original `Map` reference ... the `constructor` returns a Proxy ;-)

Reply:
Ahhh, the proxy passes the `this[PRIVATE]` getter back to the original object and accesses that `this`. Thanks for the explanation 🙂